distribution_test.go

```diff
@@ -463,12 +464,97 @@ func TestBundlePathMappingUsingSpec(t *testing.T) {
 	cleanDistributionTest(t)
 }
 
+func TestReleaseBundleCreateDetailedSummary(t *testing.T) {
+	initDistributionTest(t)
+
+	// Upload files
+	specFile, err := tests.CreateSpec(tests.DistributionUploadSpecB)
+	assert.NoError(t, err)
+	runRt(t, "u", "--spec="+specFile)
+
+	// Check error: The --detailed-summary option can't be used without --sign
+	runRbWithError(t, "rbc", tests.BundleName, bundleVersion, tests.DistRepo1+"/data/b2.in", "--detailed-summary")
```

Comment:
This command prints a long "usage" error during tests. We'd prefer to avoid this kind of output:
<img width="1246" alt="image" src="https://user-images.githubusercontent.com/11367982/117995548-166b1380-b34a-11eb-9ca2-9258a2fce7cf.png">

Please remove this test case or find a way to avoid this output.
Same for the other new tests.